apps/backoffice-v2/src/domains/business-reports/hooks/mutations/useCreateBusinessReportMutation/useCreateBusinessReportMutation.tsx

```diff
@@ -51,6 +52,12 @@ export const useCreateBusinessReportMutation = ({
       onSuccess?.(data);
     },
     onError: error => {
+      if (error instanceof HttpError && error.code === 400) {
```

Comment:
Should we use [HTTP error code 429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429)?

Reply:
It's not related... 429 is for rate limit errors.

Reply:
From what I understand, you could send the reason for the "too many requests" in the `message` property of the error. I'm guessing 400 or 403 serve the same purpose in that regard, only 400 is more "generic". Anyway 400 is fine